src/test/ci/song_selector.test.ts

```diff
@@ -85,7 +87,8 @@ describe("getFilteredSongList", () => {
                 const selectedArtist = { id: 208, name: "Twice" };
                 await guildPreference.setGroups([selectedArtist]);
                 const { songs } = await SongSelector.getFilteredSongList(
-                    guildPreference
+                    guildPreference,
+                    true
```

Comment:
Can we add a default value?

Reply:
Should the default be premium = true or premium = false?

Reply:
false